doc/installation/iOS.md

```diff
@@ -6,7 +6,7 @@ After setting up your [environment][1], you can follow the steps below to run th
 
 To install iOS modules (pods), we use Cocoapods (dependency manager). It is built with Ruby, so make sure it is installed on your computer.
 
-We use [bundler][2] to install our Ruby gems. You can install it with `gem install bundler`.
+We use [bundler][2] to install our Ruby gems. Check if you have the right version of Ruby, then you can install it with `gem install bundler`. You may need to use sudo.
```

Comment:
Peut-être indiquer comment vérifier la version de ruby ? Et surtout laquelle est la bonne ?

Aussi je suis pas très convaincue du "you may need to use sudo" un peu au hasard, c'est pas mal d'apporter des solutions à des vrais problèmes, là ça me laisse penser que tu en as eu besoin parfois mais on sait pas trop pourquoi